src/test/librbd/image/test_mock_ValidatePoolRequest.cc

```diff
@@ -219,5 +239,23 @@ TEST_F(TestMockImageValidatePoolRequest, OverwriteError) {
   ASSERT_EQ(-EINVAL, ctx.wait());
 }
 
+TEST_F(TestMockImageValidatePoolRequest, CreateTrashError) {
```

Comment:
I'd move `CreateTrashError` test above, between `RemoveSnapshotError` and `OverwriteError` tests so that they remain ordered by state.